tests/Extensions/ElementalAreaFieldExtension.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+
+namespace DNADesign\Elemental\Tests\Extensions;
+
+use SilverStripe\ORM\DataExtension;
+
+class ElementalAreaFieldExtension extends DataExtension
+{
+    public function onSaveInto($updatedElements, $page, $elementData)
+    {
```

Comment:
A little confusing having an empty function here?  Might be easier to just change one of the values in $updatedElements and do an assertion the bottom of ElementalAreaFieldTest to test that the onSaveInto extension method was called?

Reply:
A mock that spies on the method call is the correct way to test this. The implementation I have tests all three parameters and doesn't rely on side effects. Agree the empty method call is weird, but `->extend()` does a `method_exists()` call before invoking anything, so there has to be something there. https://github.com/silverstripe/silverstripe-framework/blob/4/src/Core/Extensible.php#L465